Userland/Utilities/mv.cpp

```diff
@@ -8,17 +8,16 @@
 #include <AK/String.h>
 #include <LibCore/ArgsParser.h>
 #include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
 #include <stdio.h>
 #include <string.h>
 #include <sys/stat.h>
 #include <unistd.h>
 
-int main(int argc, char** argv)
+ErrorOr<int> serenity_main(Main::Arguments args)
 {
-    if (pledge("stdio rpath wpath cpath fattr", nullptr) < 0) {
-        perror("pledge");
-        return 1;
-    }
+    TRY(Core::System::pledge("stdio rpath wpath cpath fattr", nullptr));
```

Comment:
The last argument is optional in the `Core::System::pledge(..)` wrapper, since it's a c++ function can you fix that here the other things you've ported in this PR?